### PR TITLE
Create artwork grid responsive wrapper

### PR DESCRIPTION
**Pull request by zephraph**
This creates a wrapper for the ArtworkGrid to make it easier to use in the Styleguide for now. Also wires it up on the Artwork page.

### CODE REVIEW

File: src/Styleguide/Components/ArtworkGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Responsive } from "../Utils/Responsive"
+import LegacyArtworkGrid from "Components/ArtworkGrid"
+
+export const ArtworkGrid = props => (
+  <Responsive>
+    {({ xs, sm }) => (
+      <LegacyArtworkGrid columnCount={xs ? 2 : sm ? 3 : 4} {...props} />
```
Comment (damassi) [inline src/Styleguide/Components/ArtworkGrid.tsx]: While this nested ternary reads pretty well right here would you mind breaking this out a bit into a normal if / else conditional or switch? These are hard to unpack when returning to code after being away for a while.

**Comment by zephraph** [comment]
Doesn't look like it. Artwork page `n - 1`.

**Comment by zephraph** [comment]
I updated the columns in the overview section of the artist page. Updated the ArtworkGrid placeholder component to have a `Full` version. Not super happy with that, but it does get us a little closer on the artwork page.

**Comment by damassi** [comment]
👍

**Pull request merged**